url.js

```diff
@@ -14,12 +14,14 @@ window.addEventListener ("load", changeTitle, false);
 function setExtraOptions(item) {
         protocolEnabled = item.protocol;
         pathEnabled = item.path;
+        delimiterStr = item.delimiter;
 }
 
 function prepHostname() {
+    delimiter = (delimiterStr ? delimiterStr : "");
     protocol = (protocolEnabled ? `${window.location.protocol}//` : "");
     path = (pathEnabled ? window.location.pathname : "");
-    return ` - ${protocol}${window.location.hostname}${path}`;
+    return `${delimiter}${protocol}${window.location.hostname}${path}`;
```

Comment:
We should keep the spaces around `${delimiter}` so users only have to input the characters they want as the delimiter

Reply:
Actually, disregard this one. I see the use-cases where delimiters wouldn't need spaces.